wire/internal/wire/wire_test.go

```diff
@@ -488,7 +488,7 @@ type dirInfo struct {
 
 func (d dirInfo) Name() string       { return d.name }
 func (d dirInfo) Size() int64        { return 0 }
-func (d dirInfo) Mode() os.FileMode  { return os.ModeDir | 0777 }
+func (d dirInfo) Mode() os.FileMode  { return os.ModeDir | os.ModePerm }
```

Comment:
Hmm.. I guess this is the only place where you might want to use `os.ModePerm`, since it looks like `0777` is used as a perm bits mask (purpose that `os.ModePerm` serves; it's a mask, not a 0777 perm value).